jwst/assign_mtwcs/assign_mtwcs_step.py

```diff
@@ -38,9 +38,13 @@ def process(self, input_lib):
         if not isinstance(input_lib, ModelLibrary):
             try:
                 input_lib = ModelLibrary(input_lib)
-            except Exception:
+            except Exception as err:
```

Comment:
I know you didn't write this pattern initially, but what kind(s) of exception is being caught here?  Should this be more specific?